Language/Functions/Bits and Bytes/bit.adoc

```diff
@@ -44,6 +44,32 @@ The value of the bit.
 --
 
 [float]
+=== Example Code
+// Describe what the example code is all about and add relevant code   ►►►►► THIS SECTION IS MANDATORY ◄◄◄◄◄
+Prints the output of `bit(n)` for n=0 to n=10 to the serial monitor. Values returned are 1 2 4 8 16 till 1024.
+[source,arduino]
+----
+void setup() {
+Serial.begin(9600); 
+}
+
+void loop() {
+for(int n=0;n<=10;n++)  //loop from n=0 till 10
+  {
+  Serial.print(" ");    //space between numbers, for readability
+  Serial.print(bit(n)); //print the value of bit(n)
+  delay(1000);
```

Comment:
```suggestion
  //loop from n=0 till 10
  for (int n = 0; n <= 10; n++) {
    Serial.print(" ");     //space between numbers, for readability
    Serial.print(bit(n));  //print the value of bit(n)
    delay(1000);
```

Use the standard Arduino code formatting style, as implemented by the default configuration of the Arduino IDE "**Auto Format**" feature.